pegasus-spark-analyser/src/main/java/com/xiaomi/infra/pegasus/spark/analyser/ColdBackupLoader.java

```diff
@@ -49,24 +52,28 @@ public ColdBackupLoader(ColdBackupConfig config) throws PegasusSparkException {
     LOG.info("init fds default config and get the data urls");
   }
 
-  @Override
-  public ColdBackupConfig getConfig() {
-    return coldBackupConfig;
-  }
-
   @Override
   public int getPartitionCount() {
     return partitionCount;
   }
 
   @Override
-  public Map<Integer, String> getCheckpointUrls() {
-    return checkpointUrls;
+  public ColdBackupConfig getConfig() {
```

Comment:
`public ColdBackupConfig getConfig() {`

Reply:
`final` method means the method don't be overload(different from c++ const), the method is abstract in interface, it should't be `final`